0.pre-preprocess_images/scripts/change_file_names.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Replace the strings to rename the file names
+
+# In[ ]:
+
+
+import pathlib
+
+# In[ ]:
+
+
+# set path to the input files
+input_path = pathlib.Path("../../data/00.raw_images_ims/").resolve()
+
+# set path to the output files
+output_path = pathlib.Path("../../data/0.raw_images_tiff/").resolve()
+
+# create output path if it does not exist
+output_path.mkdir(parents=True, exist_ok=True)
```

Comment:
Is this the same directory you are already creating in the `sh` file? Recommend only having one statement to create the directory since I am not sure if something will break or cause errors. Also might fall under DNR.

Reply:
addressed